<launch>
  <!-- Start Astra camera driver -->
  <include file="$(find astra_launch)/launch/astrapro.launch" />  

  <!-- Start PointCloud Fast Filter -->
  <node name="astra_people" pkg="pff_ped" type="astra_people" output="screen">
    <param name="sensorheight" value="0.60"/>     <!-- Default 0.60 (60cm) -->
    <param name="camera_fov" value="60"/>         <!-- Default 60 (60°) -->
    <param name="resolution" value="0.40"/>        <!-- Default 0.40° -->
    <param name="legs_begin" value="0.50"/> <!-- Default 0.50 (50cm) -->
    <param name="legs_end" value="0.60"/> <!-- Default 0.60 (60cm) -->
    <param name="trunk_begin" value="1.00"/> <!-- Default 1.00 (100cm) -->
    <param name="trunk_end" value="1.10"/> <!-- Default 1.10 (110cm) -->
  </node>  

  <!-- Start rviz -->
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find pff_ped)/rviz_cfg/astra_people.rviz" />

</launch>
